docs/known-issues.md

```diff
@@ -11,3 +11,4 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
 | Issue | Date added | Ticket | Workaround? | Fixed? |
 |-------|-------------------|--------|-------------|-------|
 | The Unity editor randomly crashes on MacOS when using ECS preview packages. This is not caused by the SpatialOS GDK. | 2018/07/25 | 1064084 | Simply retry. | No|
+| IL2CPP does not currently work on Mac as it will crash upon starting the game. This is not caused by the SpatialOS GDK. | 2018/08/31 | 1076596 | Use the Mono compiler. | No|
```

Comment:
Is this reported to Unity?

Reply:
yep, created a bug report. The ticket number stands for the bug report ticket with unity

Reply:
Is it worth saying in the column that 1076596 is a Unity ticket for future-self and any users to follow?

"...as it will crash upon starting the game."
->
"...as it crashes upon starting the game." 

"Will" is OK (not technically wrong) but we don't need it and the text is stronger without it - 'will' sort of reads like a prophecy.

Reply:
we could change the header from `Ticket` to `Unity Ticket`?